### PR TITLE
Add Airbnb js style guide enforcing pre commit git hook

### PR DESCRIPTION
**Pull request by WhiteHatArpit**
[#4] 
Do `npm install` to set up the pre commit hook.

Additional: Use eslint plugin with Airbnb config on your local ide/editor.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "main": "./main.js",
+  "scripts": {
+    "start": "electron .",
+    "postinstall": "for file in `ls .git-hooks`; do ln -sf ../../.git-hooks/${file} .git/hooks/${file}; done"
+  },
+  "devDependencies": {
+    "electron-prebuilt": "^0.36.8",
+    "eslint": "^2.2.0",
+    "eslint-config-airbnb": "^6.0.2",
+    "eslint-plugin-react": "^4.1.0"
```
Comment (ketanbhatt) [inline package.json:13]: Why do we need react plugin?

Comment (WhiteHatArpit) [inline package.json:13]: React plugin is an Airbnb's dependency. They might resolve this in future release or on dev branch.

Comment (ketanbhatt) [inline package.json:13]: :+1:

**Comment by ketanbhatt** [comment]
If you confirm this to work, won't test this because it is trivial.

**Comment by WhiteHatArpit** [comment]
screenshot from 2016-02-25 12 08 25
All set.

**Pull request merged**